src/types/message.ts

```diff
@@ -280,9 +289,11 @@ export function isBadMessage(message: Message): boolean {
 /**
  * All types of messages sent to and from the GCS.
  */
-export type Message = StartMessage | AddMissionMessage | PauseMessage | ResumeMessage | StopMessage
-| ConnectionAckMessage | UpdateMessage | POIMessage | CompleteMessage | ConnectMessage
-| AcknowledgementMessage | BadMessage;
+
+export type Message = StartMessage | AddMissionMessage
+| PauseMessage | ResumeMessage | StopMessage | GeofenceMessage | ConnectionAckMessage |
```

Comment:
lets keep the format the same here and have a `|` in the beginning and maximize the number of messages we can put in a line